src/Components/Web.JS/package.json

```diff
@@ -24,18 +24,17 @@
     "@typescript-eslint/parser": "^1.13.0",
     "eslint": "^5.16.0",
     "jest": "^24.9.0",
-    "rimraf": "^2.7.1",
+    "rimraf": "^3.0.2",
     "ts-jest": "^24.3.0",
     "ts-loader": "^4.4.1",
     "typescript": "^3.8.3",
-    "webpack": "^4.42.1",
-    "webpack-cli": "^3.3.11"
+    "webpack": "^4.46.0",
+    "webpack-cli": "^3.3.12"
   },
   "resolutions": {
     "**/set-value": "^2.0.1",
-    "bl": ">=2.2.1",
+    "chokidar": ">=3.5.3",
     "lodash": ">=4.17.21",
-    "tar": ">=4.4.18",
```

Comment:
If validation fails again, will probably revert or reduce the `webpack` version bump. We're using an older version in release/5.0 too.